docs/targets/solana.rst

```diff
@@ -178,6 +179,26 @@ bool ``is_writable``
 bool ``is_signer``
     Can the callee assume this account signed the transaction
 
+
+Builtin create_program_address
+++++++++++++++++++++++++++++++
+
+This function returns the program derived address for a program address, and
+some seeds. See the solana documentation on
```

Comment:
I understand what you mean by seeds, but writing `some seeds` has a strange connotation, because it sounds like one can write anything as a seed. What about writing `and the provided seeds`?

Reply:
Good point, changed